tests/find/gen.py

```diff
@@ -6,6 +8,19 @@ def crop_image(img, w, h):
     return img_crop
 
 
+def wrapper():
+    start = time.time()
+    gen("white", 350, 350, 'TimesNewRoman', 32, "File")
+    seconds = int(time.time() - start)
+    seconds = seconds % (24 * 3600)
+    hours = seconds // 3600
+    seconds %= 3600
+    minutes = seconds // 60
+    seconds %= 60
+    with open('tests/find/result.txt', 'a') as file:
```

Comment:
It is better way not to use file output for tests. Tests should use stdout for output and metrics, and stderr for error reporting